src/filters/util.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+angular.module('openshiftCommonUI')
+  .filter("toArray", function() {
```

Comment:
I know this is existing code, but probably should just be

```js
.filter('toArray', function() {
  return _.toArray;
});
```

Not for this PR, though.